server/shadow/shadow.c

```diff
@@ -73,6 +73,8 @@ int main(int argc, char** argv)
 		  "NTLM SAM file for NLA authentication" },
 		{ "keytab", COMMAND_LINE_VALUE_REQUIRED, "<file>", NULL, NULL, -1, NULL,
 		  "Kerberos keytab file for NLA authentication" },
+		{ "ccache", COMMAND_LINE_VALUE_REQUIRED, "<file>", NULL, NULL, -1, NULL,
+		  "Kerberos ccache file for NLA authentication" },
 		{ "gfx-progressive", COMMAND_LINE_VALUE_BOOL, NULL, BoolValueTrue, NULL, -1, NULL,
```

Comment:
I'd suggest to change that to `Kerberos host ccache file for NLA authentication`